CHANGELOG.md

```diff
@@ -18,92 +18,91 @@
 
 ### Changed
 
-* Improvement on the doc, notably warning that only XPath 1.0 queries are handled.
+- Improvement on the doc, notably warning that only XPath 1.0 queries are handled.
```

Comment:
As mentioned in our [contribution guidelines](https://github.com/kbrw#code-formatting), and while specific to elixir formatting, we're not keen on accepting file-wide non-functionnal changes that don't bring anything. In this case, changing the list prefix by another one when it is [valid Github markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists) feels like an unneeded change.